spaces/tests.py

```diff
@@ -228,6 +228,46 @@ def test_district_view(self):
         })
         self.assertEqual(response.status_code, 200)
 
+class MenuPageTest(TestCase):
```

Comment:
```suggestion
class CategoryTest(TestCase):
```
Page가 아닌 자원(View) 단위로 생각해 주세요.

Reply:
수정했습니다!